GoogleMeetup/src/main/webapp/script.js

```diff
@@ -315,6 +301,168 @@ function initAutocomplete() {
 
     // Add the autocomplete container to dropdown div so menu doesn't disappear
     setTimeout(function() {
-        document.getElementById("addPerson").prepend(document.getElementsByClassName("pac-container")[0]);
+        document.getElementById("addPersonContainer").prepend(document.getElementsByClassName("pac-container")[0]);
     }, 300);
 }
+
+
+// Adding a new meetup attendee 
+function addPerson() {
+    // Create new attendee dict
+    var address = document.getElementById("address").value;
+    var transport = document.getElementById("transport").value;
+    var newAttendee = {
+        "address": address,
+        "transport": transport
+    };
+
+    // Add to attendees list 
+    attendees.push(newAttendee);
+
+    // Add marker to the map and recenter
+    getLatLng(newAttendee);
+
+    // Reset the form inputs
+    document.querySelector('#addPerson').reset();
+}
+
+
+// Finding the meetup 
+function findMeetup() {
+    if (!validateInput()) {
+        return;
+    }
+
+    maxTravelTime = document.getElementsByClassName('slider')[0].value;
+
+    drawPaths();
+
+    alert("Not Finished Yet!");
+}
+
+
+function validateInput() {
+    if (attendees.length == 0) {
+        alert("You must add at least one person");
+        return false;
+    } if (destinationType == null) {
+        alert("You must select a destination type");
+        return false;
+    }
+    
+    return true;
+}
+
+
+function drawPaths(){
```

Comment:
In general, it's easier to test functions if you inject the required data rather than relying on global variables.

In this instance, you could inject the attendees and the destination.